repository/src/main/resources/alfresco/subsystems/Search/common-search-context.xml

```diff
@@ -105,8 +105,35 @@
             <ref bean="metadataQueryIndexesCheck2" />
         </property>
     </bean>
```

Comment:
Similar change needed in https://github.com/Alfresco/alfresco-enterprise-repo/blob/master/repository/src/main/resources/alfresco/subsystems/Search/common-search-enterprise-context.xml#L31-L55